scripts/cbmc_json_parser.py

```diff
@@ -273,6 +275,20 @@ def extract_solver_information(cbmc_response_json_array):
 
     return properties, solver_information
 
+def resolve_unknown_location_checks(properties):
```

Comment:
"resolve" is a bit misleading. Perhaps use "update" or "modify"?

I would also use "undefined_function_checks" instead of "unknown_location_checks".